dbhydra/dbhydra_core.py

```diff
@@ -1790,7 +1852,7 @@ def insert_from_df(self, df, batch=1, try_mode=False, debug_mode=False):
 
         # handling nan values -> change to NULL TODO
         for column in list(df.columns):
-            df.loc[pd.isna(df[column]), column] = "NULL"
+            df.loc[pd.isna(df[column]), column] = None
```

Comment:
I am not sure here

Reply:
I had issues proper datatype casting using `NULL` values written as a string in excel files. On the other hand, keeping `None` values mapped nicely to empty cell (in excel), and is actually distinguishable from a `NULL` string.

So far in my previous tests it worked well.